src/tool_writeout.c

```diff
@@ -193,6 +198,51 @@ static int writeString(FILE *stream, const struct writeoutvar *wovar,
   }
   else {
     switch(wovar->id) {
+    case VAR_CERT:
+      if(per->certinfo) {
+        int i;
+        bool error = FALSE;
```

Comment:
A `goto` might be cleaner for exiting a nested loop

Reply:
I like gotos when cleaning up and getting out of a function, but not to just abort a loop.